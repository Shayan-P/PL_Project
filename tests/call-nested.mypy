def f(x=1, y=2):
    return x + y;
    ;

print(f(f(1, 2)));
print(f([1, 2, 3], [4, 5, 6]));
