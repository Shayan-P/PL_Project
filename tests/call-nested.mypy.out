(num-val 5)
(list-val
 (list
  (num-val 1)
  (num-val 2)
  (num-val 3)
  (num-val 4)
  (num-val 5)
  (num-val 6)))
'finished
